code/modules/mob/mob_movespeed.dm

```diff
@@ -108,7 +108,7 @@ Key procs
 				conflict_tracker[conflict] = amt
 			else
 				continue
-		. *= amt
+		. += amt

```

Comment:
dunno if you want this changed just yet, lemme know

Reply:
Yeah this is fine, I'll be messing with movespeed in a bit